packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -123,13 +126,15 @@ export async function runBuildAndSubmitAsync(
     (await EasJsonUtils.getCliConfigAsync(easJsonAccessor)) ?? {};
 
   const platforms = toPlatforms(flags.requestedPlatform);
-  const buildProfiles = await getProfilesAsync({
-    type: 'build',
-    easJsonAccessor,
-    platforms,
-    profileName: flags.profile ?? undefined,
-    projectDir,
-  });
+  const buildProfiles =
+    buildProfilesOverride ??
```

Comment:
Just curious (I'm not super familiar with build profiles), from a UX perspective should we require them to specify/choose an existing dev-client-enabled profile instead of creating a virtual profile for them? If not, are there any side effects of creating this "virtual" profile for them?

Reply:
Hmm, I think what you suggest here might be a better option 🤔 The Current implementation should "magically" work well out of the box for users, but I think it limits the customization of things (like selecting the `bun` version). I believe that using explicit profiles can also have the benefit of people having a good idea about what settings we apply under the hood for these builds 🤔 What do you think?